### PR TITLE
Feature/mixed elastic

### PR DESCRIPTION
**Pull request by whaeck**
Mixed mode elastic for thermal scattering in THERMR and ACER (new ENDF/B-VIII.1 feature).

Plotting in ACER was updated accordingly too.

I've also added 4 new tests (tests 67-70) for the following cases:
- inelastic only
- inelastic + coherent elastic
- inelastic + incoherent elastic
- inelastic + coherent & incoherent elastic

The first three give the same results for older versions of NJOY so we still produce the same results for older evaluations.

The older two PRs should be merged first since this one uses some of the changes from those olders ones too.

### CODE REVIEW

**Comment by marquezj** [comment]
@whaeck Great!

I think this will come in handy here: I prepared a couple of Google Colab notebooks that installs this version and runs mixed-elastic materials from the [NJOY+NCrystal Library](https://github.com/highness-eu/NJOY-NCrystal-Library).

- [Aluminum](https://colab.research.google.com/drive/1X8esQH59T2zj2oxBmaIRQOBp-8iFKfHV?usp=sharing)
- [Nickel](https://colab.research.google.com/drive/1MTE3K2qPmzISq0RNIs1r1OQKiJ31vdrx?usp=sharing)
- [Magnesium Deuteride](https://colab.research.google.com/drive/1W0qmR0yoeedR68kfD85DUFog91k6G1Xb?usp=sharing)
- [Uranium Fluoride](https://colab.research.google.com/drive/1W0qmR0yoeedR68kfD85DUFog91k6G1Xb?usp=sharing)

Some notes:

- ACE creation works, and the files can be read with [our modified version of OpenMC](https://github.com/highness-eu/openmc/tree/njoy_ncrystal).

- ACE plotting works for Al, but fails for Ni, MgD2 and UF6. VIEWER produces NaN values in the Postscript output. I will try to find the bug.

- ACER still has debugging print statements.

**Comment by whaeck** [comment]
@marquezj Should be fixed now, it happened when plotting mubar. When looping over the incident energy values, I used the wrong number of incident energy values for the incoherent elastic part (nee instead of neei). The plot at 293.6 now renders properly. It did not happen when I first tried it at 400K though.

**Comment by marquezj** [comment]
@whaeck Wonderful. Thanks.

File: src/aceth.f90 [hunk 1]
```diff
@@ -648,76 +670,157 @@ subroutine thrlod(nin,matd,tempd,tname,suff,izn,&
...
-         do i=1,nee
-            xss(i+itce)=scr(1+2*(i-1))/emev
-            xss(i+nee+itce)=scr(2+2*(i-1))/emev/nmix
-         enddo
+   !--read first elastic block - coherent elastic
+   nee=0
+   nea=0
+   read(nin) nee,nea
+
+print*, "coherent elastic ", nee, nea
```
[the remark this answers is not in the document]
Comment (whaeck) [reply src/aceth.f90:703]: I completely forgotten about that. Just a sec ...

**Review by nathangibson14 — APPROVED** [review]
I'm clicking "approve" now, but we should make sure the CI runs before getting too carried away merging this into other branches.

**Pull request merged**